lg_media/scripts/image_checker.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import json
+import rospy
+import subprocess
+from lg_common.srv import USCSMessage
+
+PGREP_CMD = ['pgrep', '-a', 'feh']
+EGREP_CMD = ['egrep', '-o', 'http?://[^ ]+']
+PARTIAL_KILLER_CMD = ['pkill', '-9', '-f']
+IMAGE_PROCS_TO_KILL = ['image_viewer.py', 'feh', 'pqiv']
+LOOP_TIMEOUT = 5
+
+
+class ImageChecker():
+    def __init__(self, last_uscs_service):
+        self.last_uscs_service = last_uscs_service
+        self.last_uscs = ''
+        self._keep_alive()
+
+    def _keep_alive(self):
+        while not rospy.is_shutdown():
+            assets_to_remove = []
+            current_image_assets = []
+            self.last_uscs = json.loads(self.last_uscs_service().message)
+            for window in self.last_uscs['windows']:
+                if window['activity'] == 'image':
+                    current_image_assets.append(window['assets'][0])
+
+            feh_assets = self._get_feh_assets()
+            if feh_assets:
+                for feh_asset in feh_assets:
+                    if feh_asset not in current_image_assets:
+                        assets_to_remove.append(feh_asset)
+
+                if assets_to_remove:
+                    if self.last_uscs == json.loads(self.last_uscs_service().message):
+                        for image_proc in image_procs_to_kill:
+                            subprocess.call(PARTIAL_KILLER_CMD + [image_proc])
+            rospy.sleep(LOOP_TIMEOUT)
+
+    def _get_feh_assets(self):
+        feh_assets = ''
```

Comment:
The way this works, you are assigning feh_assets to an empty string then returning a list if there were any assets.  If you initialize and empty list here and guarantee that the return value of this method is a list, it makes your life easier.  Then you don't have to check `if feh_assets` on L30, you just start iterating over the list.